app/requirements.txt

```diff
@@ -0,0 +1,21 @@
+fastapi
```

Comment:
Please make sure to include `fastapi[standard]` as outlined in the _latest_ `README.md` document.